packages/@react-aria/select/src/useSelect.ts

```diff
@@ -68,6 +68,27 @@ export function useSelect<T>(props: AriaSelectOptions<T>, state: SelectState<T>,
     ref
   );
 
+  let onKeyDown = (e: KeyboardEvent) => {
+    if (keyboardDelegate) {
```

Comment:
Did some testing locally, noticed that this didn't work in the `useSelect` example. Should this be `keyboardDelegate` or should it use the `delegate` defined above? I _think_ we don't need the if statement and the other `keyboardDelegates` can be changed to `delegate`, lemme know if you encountered something otherwise

Reply:
yeah that works - why are those aria examples not part of the storybook? I see only `useMove` has a story in there.

Reply:
For the most part, the storybook is primarily for testing the React Spectrum components. There are a couple exceptions where there are aria specific stories as you noticed but those are usually only added if the spectrum component that uses said hooks doesn't exist yet.